warema_wms/wms_controller.py

```diff
@@ -69,6 +70,7 @@ def _increment(self):
         self.initial_ts += 1
         return res
 
+
```

Comment:
```suggestion
```

PEP8 says: Only one empty line between methods.

Reply:
had to google who this PEP8 is, but he's probably right :)

and thanks for the hint to this style guide, will have some looks more